builds/azure-pipelines-release.yml

```diff
@@ -130,7 +130,7 @@ jobs:
 
   - task: MSBuild@1
```

Comment:
Maybe we can try vsbuild task instead of msbuild task?
 https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/vsbuild-v1?view=azure-pipelines

Reply:
Looks like the build error happens locally as well so I don't know if that'll help just yet.